diagnostic.md

```diff
@@ -7,40 +7,41 @@ What command do we use to do that? Use the **correct** URL from your fork of
 this repository in your answer.
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+Git clone https://github.com/alzateja/git-diagnostic.git
 ```
 
 2.What do you do after cloning a repository, before you start making any
 changes/additions?
 
-<!-- Remove this comment and place your answer here. -->
+I changed direction into the new git-diagnostic folder using "cd git-diagnostic"
```

Comment:
Hey this is on the right track but we were specifically looking for creating a new branch that you could work on without affecting the master branch.